electron-sign/README.md

```
@@ -347,98 +427,300 @@ npx electron . --enable-logging --verbose
+
+# Verify code signing setup (macOS)
+codesign --verify --deep --strict --verbose=2 path/to/Giddh.app
+```
+
+## S3 Bucket Structure & Environment Detection
+
+### Dynamic Path Structure
+
+The application uses a dynamic S3 path structure based on the environment:
+
+```
```

markdownlint-cli2 (0.20.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)